src/pages/costModels/createCostModelWizard/markup.tsx

```diff
@@ -1,79 +1,115 @@
-                    helperTextInvalid={t('cost_models_wizard.markup.invalid_markup_text')}
-                    validated={isValid ? 'default' : 'error'}
-                  >
-                    <InputGroup style={{ width: '150px' }}>
-                      <TextInput
-                        type="text"
-                        id="markup"
-                        name="markup"
-                        value={markup}
-                        onBlur={() => {
-                          this.setState({ isValid: !isNaN(Number(markup)) });
-                        }}
-                        onChange={(value: string) => {
-                          onMarkupChange(value);
-                          if (value !== '-') {
-                            this.setState({ isValid: !isNaN(Number(value)) });
-                          }
-                        }}
-                        validated={isValid ? 'default' : 'error'}
-                      />
-                      <InputGroupText style={{ borderLeft: '0' }}>%</InputGroupText>
-                    </InputGroup>
-                  </FormGroup>
+                  <Grid hasGutter>
+                    <GridItem lg={8} id="refSign">
+                      <FormGroup isInline fieldId="markup-or-discount">
+                        <div style={styles.radioAlign}>
+                          <Radio
+                            isChecked={!isDiscount}
+                            name="discount"
+                            label={t('cost_models_details.markup_plus')}
+                            aria-label={t('cost_models_details.markup_plus')}
+                            id="markup"
+                            value="false" // "+"
+                            onChange={handleSignChange}
+                          />
+                        </div>
+                        <div style={styles.radioAlign}>
+                          <Radio
+                            isChecked={isDiscount}
+                            name="discount"
+                            label={t('cost_models_details.discount_minus')}
+                            aria-label={t('cost_models_details.discount_minus')}
+                            id="discount"
+                            value="true" // '-'
+                            onChange={handleSignChange}
+                          />
+                        </div>
+                      </FormGroup>
+                    </GridItem>
+                    <GridItem lg={4} id="refMarkup">
+                      <FormGroup
+                        isInline
+                        fieldId="rate"
+                        helperTextInvalid={t('cost_models_wizard.markup.invalid_markup_text')}
+                      >
+                        <InputGroup style={{ width: '150px' }}>
+                          <InputGroupText style={styles.sign}>{isDiscount ? '-' : '+'}</InputGroupText>
+                          <TextInput
+                            style={{ borderLeft: '0' }}
```

Comment:
It would be best not to introduce inline styles -- add them to `costCalc.styles.ts`. Same goes for `updateMarkupDialog.tsx` -- mentioned that in a previous review.

Reply:
Done